yarn-project/simulator/src/client/private_execution.test.ts

```diff
@@ -216,6 +217,11 @@ describe('Private Execution test suite', () => {
     return trees[name];
   };
 
+  const computeNoteHash = (note: Note, storageSlot: Fr) => {
+    // We're assuming here that the note hash function is the default one injected by the #[note] macro.
+    return poseidon2HashWithSeparator([...note.items, storageSlot], GeneratorIndex.NOTE_HASH);
+  };
```

Comment:
This test used to call `compute_note_hash_and...` to check that the simulation side effects were exactly what was expected. Fully replicating this would require also computing nullifiers, which would've been fragile (as can be seen here in the comment re. note hashing), and given that the original test seemed way overkill and overfit, I just got rid of those checks.